designer/server/src/common/nunjucks/context/index.js

```diff
@@ -29,8 +30,10 @@ export async function context(request) {
   }
 
   const credentials = request ? await getUserSession(request) : undefined
+  const envTag = environmentTagHelper(env, isTest, isProduction)
```

Comment:
Seems a bit excessive to pass in both the env tag and the flags? Wouldn't `env` cover this on its own?